backend/data/migrations/20190306223832_league_01.js

```diff
@@ -0,0 +1,77 @@
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable('league', league => {
+      league.increments();
+      league
+      .string('name')
+      .notNullable();
+     league
```

Comment:
You might want to change styling here to make it consistent with above, also I would recommend a tab before the methods to make this easier to read.